coderd/database/dbauthz/system.go

```diff
@@ -0,0 +1,194 @@
+package dbauthz
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/coder/coder/coderd/database"
+)
+
+// TODO: All these system functions should have rbac objects created to allow
```

Comment:
Just so I understand, all of these are temporary while we don't have objects associated, right?

Reply:
The current pass through implementation is temporary yes. We want to follow this up in another PR.

Current solutions include ideas such as `db.Sudo()`, creating new rbac resources, or not using rego authz at all.

----

But until we implement that solution, we kept them in their own file `system.go` so we can easily identify which methods need to be addressed. Remember that this keeps parity with `main` since there are currently no rbac checks for these methods in `main`.